### PR TITLE
Load default configs from .swfrc

### PR DESCRIPTION
**Pull request by ziyunli**
https://trello.com/c/ZA55cNgY/276-aws-swf-library-take-out-defaults

### CODE REVIEW

**Comment by ziyunli** [comment]
ping @ktonon

**Comment by ktonon** [comment]
Yeah, sorry, I wasn't clear in the card on trello. These values should not be set at all. When defining an activity on AWS SWF, the default values can be defined there.

File: lib/decision-response.js [hunk 1]
```diff
@@ -200,16 +201,16 @@ DecisionResponse.prototype = {
...
-        ta.scheduleToCloseTimeout = scheduleAttributes.scheduleToCloseTimeout || "360";
+        ta.scheduleToCloseTimeout = scheduleAttributes.scheduleToCloseTimeout || config.scheduleToCloseTimeout;
       }
       if (!ta.startToCloseTimeout) {
-        ta.startToCloseTimeout = scheduleAttributes.startToCloseTimeout || "300";
+        ta.startToCloseTimeout = scheduleAttributes.startToCloseTimeout || config.startToCloseTimeout;
       }
       if (!ta.heartbeatTimeout) {
-        ta.heartbeatTimeout = scheduleAttributes.heartbeatTimeout || "60";
+        ta.heartbeatTimeout = scheduleAttributes.heartbeatTimeout || config.heartbeatTimeout;
```
Comment (ktonon) [inline lib/decision-response.js]: This whole section should just be removed

**Review by ktonon — CHANGES_REQUESTED** [review]

**Comment by ziyunli** [comment]
Activity types and workflow types with all required defaults are created under the domain `content-service-staging` using SWF configs generated by https://github.com/Brightspace/d2l-content-service-swf-decider/pull/20

**Comment by ziyunli** [comment]
ping @ktonon I think this PR should be safe to merge now, since we've already created types with all required defaults on AWS. Also, this shouldn't be used by decider until we change the aws-swf's git sha in the decider's package.json

**Pull request merged**